src/App.js

```diff
@@ -1,12 +1,11 @@
-//import logo from './logo.svg';
 import './App.css';
+import CreateTabs from "./CreateTabs.js"
 
 function App() {
   return (
     <div className="App">
       <h1>Echo's Portfolio Website </h1>
```

Comment:
Do you want this at the top of your website lol, it looks cheeesy

Reply:
You're making me feel insecure